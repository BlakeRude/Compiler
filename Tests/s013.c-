// C-S21
// testing static variables

int goad:666;
main()
{
    static int load;
    {
        static int goad:777;
        output(goad);
    }
    output(goad);
    outnl();
}
