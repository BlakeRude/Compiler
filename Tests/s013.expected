WARNING(7): The variable 'load' seems not to be used.
WARNING(7): The variable 'load-1' seems not to be used.
Number of warnings: 2
Number of errors: 0
------------------------------------
Loading file: s013.tm
777 666
Bye.
